src/WebJobs.Extensions.DurableTask.Analyzers/SyntaxNodeUtils.cs

```diff
@@ -196,6 +196,8 @@ private static bool TryGetAttribute(SyntaxNode node, string attributeName, out S
 
         internal static bool TryGetInvocationExpression(SyntaxNode node, out SyntaxNode invocationExpression) => TryGetContainingSyntaxNode(node, SyntaxKind.InvocationExpression, out invocationExpression);
 
+        internal static bool TryGetClassDeclaration(SyntaxNode node, out SyntaxNode classDeclaration) => TryGetContainingSyntaxNode(node, SyntaxKind.ClassDeclaration, out classDeclaration);
```

Comment:
Would it make sense to change this `out` parameter to be `ClassDeclarationSyntax` instead of `SyntaxNode`?